dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/enrollment/HibernateEnrollmentStore.java

```diff
@@ -99,22 +99,19 @@ private String buildCountEnrollmentHql(EnrollmentQueryParams params) {
 
   public List<Enrollment> getEnrollments(EnrollmentQueryParams params) {
     String hql = buildEnrollmentHql(params).getFullQuery();
-
     Query<Enrollment> query = getQuery(hql);
-
     return query.list();
   }
 
   public Page<Enrollment> getEnrollments(EnrollmentQueryParams params, PageParams pageParams) {
     String hql = buildEnrollmentHql(params).getFullQuery();
-
     Query<Enrollment> query = getQuery(hql);
+
```

Comment:
why this line break here?

Reply:
I compared the non-paginated with the paginated side by side and did not see the similarity at first. Thats why I grouped the statements.